src/HttpServer.ts

```diff
@@ -99,6 +98,31 @@ export class HttpServer {
         res.send(status);
     }
 
+    async processRootHandler(method: string, params: any) {
+        switch (method) {
+            case 'getMinGasPrice':
+                return await this.backend.getMinGasPrice();
+            case 'isCustomReplenish':
+                return await this.backend.isCustomReplenish();
+            case 'getManagerBalance':
+                return await this.backend.getManagerBalance();
+            case 'getWorkerBalance':
+                // @ts-ignore
+                return await this.backend.getWorkerBalance(...params);
+            case 'getAllHubEventsSinceLastScan':
+                return await this.backend.getAllHubEventsSinceLastScan();
+            case 'isTrustedVerifier':
+                // @ts-ignore
+                return await this.backend.isTrustedVerifier(...params);
+            case 'isReady':
+                return await this.backend.isReady();
+            default:
+                throw Error(
+                    `Implementation of method ${method} not available on backend!`
+                );
```

Comment:
Let's try to avoid switch/case. I'd suggest something like:

```suggestion
async processRootHandler(method: string, params: unknown) {
        const availableMethods = [
            'getMinGasPrice',
            'isCustomReplenish',
            'getManagerBalance',
            'getWorkerBalance',
            'getAllHubEventsSinceLastScan',
            'isTrustedVerifier',
            'isReady'
        ] as const;

        const backendMethod = availableMethods.find((m) => m === method);
        if (!backendMethod) {
            throw Error(
                `Implementation of method ${method} not available on backend!`
            );
        }

        return this.backend[backendMethod](params as never);
    }
```

Reply:
Im not sure about having the available methods outside the class, at this point I created a private const with the methods.